content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,204 @@
+| Distance Between Filter (Radius) |     | Settings for controlling per instance overrides for the distance between filter component. |     |
+|     | **Distance Between Filter Override Enabled** | When enabled, the distance between filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Bound Mode** | When performing distance checks between two instances, this setting determines whether the radius of the mesh asset is used vs a manually entered radius<br><br>Radius<br><br>MeshRadius. |     |
+|     | **Radius Min** | User defined radius for distance checks. | Float |
+| Surface Slope Alignment |     |     |     |
+|     | **Surface Slope Alignment Override Enabled** | When enabled, the slope alignment modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Surface Slope Alignment Min** | Override minimum alignment strength. | Float |
+|     | **Surface Slope Alignment Max** | Override maximum alignment strength. | Float |
+| Slope Filter |     |     |     |
+|     | **Slope Filter Override Enabled** | When enabled, the slope filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Slope Filter Min** | Override minimum slope accepted by filter. | Float |
+|     | **Slope Filter Max** | Override maximum slope accepted by filter. | Float |
+| Surface Mask Filter |     |     |     |
+|     | **Override Mode** | Controls how overrides are considered by the surface mask filter component<br><br>Disable - Overrides are completely ignored<br><br>Replace - Overrides replace those specified in the component<br><br>Extend - Overrides are added to the specified in the component. |     |
+|     | **Inclusion Tags** | A set of surface tags that dictate where vegetation can be placed. | SurfaceTagVector |
+|     | **Exclusion Tags** | A set of surface tags that dictate where vegetation can not be placed. | SurfaceTagVector |
+| Surface Mask Depth Filter |     |     |     |
+|     | **Surface Tags** | A set of surface tags used for depth comparisons. | SurfaceTagVector |
+|     | **Upper Distance Range (m)** | A range used for vertical distance comparisons against the closest point with matching surface tags. | Float |
+|     | **Lower Distance Range (m)** | A range used for vertical distance comparisons against the closest point with matching surface tags. | Float |
+
+File: [`/Gems/Vegetation/Code/Source/Descriptor.cpp`](https://github.com/o3de/o3de/blob/900bd390b9b11c83a77ac3de651dc9eb3d897270/Gems/Vegetation/Code/Include/Vegetation/Descriptor.h)
+
+
+## EBus interface
+
+### Vegetation::AreaInfoBus
+
+| Request Name | Description | Parameters | Return |
+| --- | --- | --- | --- |
+| `GetLayer` |     | None | Float |
```

Comment:
Comment in code: "to handle overlapping vegetation layers"
@mbalfour-amzn Can you further explain the use of this function?

Reply:
"Layer" and "Priority" are two different priority fields. "Layer" is a macro priority (foreground / background), and "Priority" is a micro priority within a layer. Some examples:
* Foreground / 0 is higher priority than Background / 0.
* Foreground / 10 is higher priority than Foreground / 0.
* Foreground / 0 is higher priority than Background / 10.

The purpose of the priorities is that it's possible to create multiple vegetation areas that overlap in the world. When that occurs, for the overlapped area, the highest-priority vegetation area is given the opportunity to spawn (or block) vegetation instances. The other vegetation areas are then given turns in decreasing priority order until there's no more room to spawn anything, or no more vegetation areas to process.

GetLayer() returns the value of the macro priority (the "layer")
GetPriority returns the value of the micro priority (the "priority")